service/src/main/java/de/starwit/service/impl/TemplateFileService.java

```diff
@@ -34,24 +34,8 @@
 	}
 
 	@Override
-	public TemplateFile findById(Long id) {
-		return this.templateFileRepository.findById(id).orElseThrow(() -> new EntityNotFoundException(String.valueOf(id)));
-	}
-
-	@Override
-	public List<TemplateFile> findAll() {
-		return this.templateFileRepository.findAll();
-	}
-
-	@Override
-	public TemplateFile saveOrUpdate(TemplateFile entity) throws ValidationException {
-		return this.templateFileRepository.save(entity);
-	}
-
-	@Override
-	public void delete(Long id) throws de.starwit.persistence.exception.EntityNotFoundException {
-		this.templateFileRepository.deleteById(id);
-		
+	public TemplateFileRepository getRepository() {
+		return templateFileRepository;
```

Comment:
Why do you return a whole repository here? Shouldn't only this Service execute actions on the repository?